projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -8,9 +8,10 @@
  */
 
 // Load dependencies.
-require_once dirname( __DIR__ ) . '/pluggable.php';
 require_once dirname( __DIR__ ) . '/class-scheduled-updates.php';
```

Comment:
Are any of them really needed? This file gets loaded from `class-scheduled-updates.php`.

Reply:
Tested and confirmed that both are not needed at runtime and during unit tests.